sources/packages/api/src-sql/DisbursementReceiptValues/Create-disbursement-receipt-values-table.sql

```diff
@@ -0,0 +1,35 @@
+--Create table sims.disbursement_receipt_values.
+CREATE TABLE IF NOT EXISTS sims.disbursement_receipt_values(
+    id SERIAL PRIMARY KEY,
+    disbursement_receipt_id INT NOT NULL REFERENCES sims.disbursement_receipts(id),
+    grant_type VARCHAR(6) NOT NULL,
+    grant_mount NUMERIC(5) NOT NULL,
```

Comment:
This one is tricky, I understood why it is created as a `NUMERIC(5)`, but the file is supposed to come with 2 additional decimals for the cents (that is the test file is "garbage" like '{0'). Should we maybe create `NUMERIC(7,2)`?
@abschwenker, thoughts?